shared/util/local-log.js

```diff
@@ -1,11 +1,13 @@
 // @flow
 
-export let logLocal = null
```

Comment:
we should only export const things. here i just made it such that you call setupLocalLogs() and it'll return the values. its likely an error to use logLocal before you call this function so lets just disallow that